web/src/engine/websites/MangaRaw1001.ts

```diff
@@ -0,0 +1,40 @@
+import { Tags } from '../Tags';
+import icon from './MangaRaw1001.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as MangaReader from './decorators/MangaReaderCMS';
+
+const pageScript = `
```

Comment:
I can't see anything special in this script. Any reason why using `FetchBrowser` instead of implementing `FetchPages` based on the normal `Fetch`?

Reply:
FetchBrowser? You mean because i use FetchWindowScript? 

Its just because we need the CHAPTER_ID variable for the ajax request. Which is only available inside chapter page.
It cant be inferred from resource uri.

I find it cleaner to do everything in one go than using FetchWindowScript or Regexp to get CHAPTER_ID  then perform the ajax request ourselves.

Reply:
It seemed to me like the chapter ID was in the URL, but then this assumption was probably wrong
![image](https://github.com/manga-download/haruneko/assets/27136773/19290db2-767c-444e-8a04-c42408ce0c62)

Reply:
Note: It seems the real chapter ID is available in the JSON chapter list on the manga page

![image](https://github.com/manga-download/haruneko/assets/27136773/371e0549-193c-4c5e-8f68-d32a09715300)

Reply:
Nice catch, now i wonder if this ld-json thing is reliable.